src/og/control/ruler/Ruler.js

```diff
@@ -38,6 +45,26 @@ class Ruler extends Control {
     ondeactivate() {
         this.renderer.removeNode(this._rulerScene);
     }
-}
+
+
+    createMenuBtn() {
+        let btn = elementFactory('div', { id: 'ruler-menu-btn', class: 'ruler menu-btn OFF' },
```

Comment:
need prefix **og-**

Reply:
Done. you want the prefix og- to be present in all css classes and ids? In my opinion it reduces a bit the readability of the code